bigbluebutton-html5/imports/ui/components/connection-status/modal/components/connectionStatusSummary.tsx

```
@@ -0,0 +1,196 @@
+    statusesSummary.statusCriticalOccurrences,
+  ]);
+
+  return (
+    <Styled.MetricsContainer>
+      <Styled.ColorIndicatorContainer>
+        {Object.entries(statusesMetrics).map(([status, values]) => {
+          return (
+            <Styled.ColorIndicators
+              status={status}
+              width={values.percentage}
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)